browser/installer/package-manifest.in

```diff
@@ -146,7 +146,7 @@
 @BINPATH@/@DLL_PREFIX@mozavcodec@DLL_SUFFIX@
 #endif
 @RESPATH@/browser/blocklist.xml
-#ifdef XP_WIN
+#if defined(XP_WIN) || defined(XP_LINUX)
```

Comment:
Can this be enhanced to include (edge case) FreeBSD?

A few weeks ago I set aside a note of the likely definition … don't have it handy (sorry) …